app/helpers/application_helper.rb

```diff
@@ -1,2 +1,23 @@
 module ApplicationHelper
+
+  def vote_count
+    self.votes.map {|vote| vote.value}.reduce(:+).to_s
+  end
+
+  def write_comment_count
+    self.comments.count.to_s
+  end
+
+  def write_user_name
+    self.user.username
```

Comment:
This would be better as a delegate method on a model.

`delegate :username, to: :user`